packages/design/buildFoundation.js

```diff
@@ -2,13 +2,33 @@
 /* eslint-disable @typescript-eslint/no-var-requires */
 const fs = require("fs");
 const postcss = require("postcss");
-const postcssCustomProperties = require("postcss-custom-properties");
+const postcssExtract = require("@csstools/postcss-extract");
 
 const foundation = fs.readFileSync("./foundation.css");
 
 postcss([
-  postcssCustomProperties({
-    exportTo: ["src/foundation.js"],
+  postcssExtract({
+    queries: {
+      customProperties: 'rule[selector*=":root" i] > decl[value]',
+    },
+    results: yourResults => {
+      const mappedResults = yourResults.customProperties.reduce(
+        (acc, { prop, value }) => {
+          if (!(prop in acc)) {
+            acc[prop] = value; //.trim(); //.replaceAll("\n", "");
```

Comment:
Ohhh, that was clever. I think we can only remove the comment! Good solution to not rewrite the values!

Reply:
Ahh, thanks. I missed that

Reply:
@rodrigoeidelvein Updated it